src/styles/NavButtons.scss

```diff
@@ -52,6 +55,10 @@ $header-height: 8vh;
     right: 10px;
     top: 2px;
   }
+  // Hide buttons on mobile screens
+  @media screen and (max-width: 768px) {
```

Comment:
Let's just move these into the above &-text block css. We only want the text to disappear, not the entire button.